File: Bon_plans_desktop_app/src/tn/esprit/bonplans/entity/Categorie.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package tn.esprit.bonplans.entity;

/**
 *
 * @author SadfiAmine
 */
public class Categorie {
    private int idCategorie;
    private String titre;
    private String urlPhoto;

    public Categorie() {
    }

    public Categorie(int idCategorie, String titre, String urlPhoto) {
        this.idCategorie = idCategorie;
        this.titre = titre;
        this.urlPhoto = urlPhoto;
    }

    public Categorie(String titre, String urlPhoto) {
        this.titre = titre;
        this.urlPhoto = urlPhoto;
    }

    public int getIdCategorie() {
        return idCategorie;
    }

    public void setIdCategorie(int idCategorie) {
        this.idCategorie = idCategorie;
    }

    public String getTitre() {
        return titre;
    }

    public void setTitre(String titre) {
        this.titre = titre;
    }

    public String getUrlPhoto() {
        return urlPhoto;
    }

    public void setUrlPhoto(String urlPhoto) {
        this.urlPhoto = urlPhoto;
    }
    
    
    
}

File: www/Bon_plans_web_app/Entity/Categorie.php
<?php
/**
 * Created by PhpStorm.
 * User: SadfiAmine
 * Date: 15/04/2018
 * Time: 12:14
 */

class Categorie
{
    private  $idCategorie;
    private $titre;
    private $urlphoto;

    function __construct()
    {
    }
    function __construct1($idCategorie,$titre,$urlphoto)
    {$this->idCategorie=$idCategorie;
     $this->$titre=$titre;
     $this->$urlphoto=$urlphoto;
    }
    function __construct2($titre,$urlphoto)
    {
        $this->$titre=$titre;
        $this->$urlphoto=$urlphoto;
    }


    /**
     * @return mixed
     */
    public function getIdCategorie()
    {
        return $this->idCategorie;
    }

    /**
     * @param mixed $idCategorie
     */
    public function setIdCategorie($idCategorie)
    {
        $this->idCategorie = $idCategorie;
    }

    /**
     * @return mixed
     */
    public function getTitre()
    {
        return $this->titre;
    }

    /**
     * @param mixed $titre
     */
    public function setTitre($titre)
    {
        $this->titre = $titre;
    }

    /**
     * @return mixed
     */
    public function getUrlphoto()
    {
        return $this->urlphoto;
    }

    /**
     * @param mixed $urlphoto
     */
    public function setUrlphoto($urlphoto)
    {
        $this->urlphoto = $urlphoto;
    }
}